doc/application/application.rst

```diff
@@ -675,15 +675,48 @@ following procedure:
 #. Rebuild the application normally following the steps specified
    in `Build an Application`_ above.
 
-
+.. _application_run:
 
 Run an Application
 ******************
 
-An application image can be run on real or emulated hardware. The kernel has
-built-in emulator support for QEMU. It allows you to run and test an application
-virtually, before (or in lieu of) loading and running it on actual target
-hardware.
+An application image can be run on real or emulated hardware.
+
+Most boards supported by Zephyr let you flash a compiled binary using
+the ``flash`` target to copy the binary to the board and run it.
+Follow these instructions to flash and run an application on real
+hardware:
+
+#. Open a terminal console and navigate to the application directory
```

Comment:
To clarify, I'd suggest saying, 

     Open a terminal console on your host computer, and navigate ...

Reply:
I'm just using the wording that was already in this document, but I agree that that's better. I'll make the same change below where it already appears for 'make run'.